reference/slot.md

```diff
@@ -2,8 +2,8 @@
 
 ```
 Slot {
-  SFString type     ""
-  SFNode   endPoint NULL
+  SFString type     ""     # any string
+  SFNode   endPoint NULL   # {Slot, Group (or derived), PROTO}
```

Comment:
Missing PointLight, SpotLight, Shape

Reply:
I'm tempted to put # {node, PROTO} here to be consistent with group.md.